pipelines/jobs/configurations/jdk11u_pipeline_config.groovy

```diff
@@ -89,11 +89,9 @@ class Config11 {
         ppc64Aix    : [
                 os                  : 'aix',
                 arch                : 'ppc64',
-                additionalNodeLabels: [
-                        temurin: 'xlc16&&aix710',
-                        openj9:  'xlc16&&aix715'
```

Comment:
Was the change from xlc16 to xlc13 intended for 11/17?

Reply:
No, but all machines should have both compilers installed regardless and this isn't where the compiler to use is selected.